### PR TITLE
Falk-Class Syndicate Robotics Vessel

### PR DESCRIPTION
**Pull request by spockye**
## About The Pull Request
image

image

this PR adds a new (totally Tubular) Syndicate ship! its purpose is to as the name says, Research and construct robots and mechs. 
Lore-wise, it's a "Gorlex ship with a Cybersun Compliance Officer and two cybersun specialists filling the medical and robotics roles".
 Due to the Financial situation of the Gorlex marauders, some ended up working for other factions.  this is one of those crews, serving under the thumb of Cybersun. 

crew:
1x Research director (gorlex)
1x cybersun official (cybersun)
1x cargo tech (gorlex)
1x medtech (cybersun)
1x security officer (gorlex)
2x roboticists (cybersun)
1x mechanic (gec)
2x deck assistants (gorlex)

- [x]  finish the json file
- [x] potential balance changes
- [x] possible re-naming

## Why It's Good For The Game
more faction ships = good. (also fills a currently empty niche). 
it would also open up for some good cybersun-gorlex roleplay. 

## Changelog
:cl:
add: Falk-class Robotics vessel
/:cl:

### CODE REVIEW

**Comment by spockye** [comment]
some pipes seem to have... deleted themselves

**Comment by Latentish** [comment]
By "tubular" you mean It's a tube.
Pretty tube tube tube tube tube

**Comment by spockye** [comment]
lord forgive me, for i have sinned. there's one object on this shuttle SOMEWHERE that has been var edited with something called "flicker chance", and i have no idea what it is. please spare me.

File: code/modules/jobs/job_types/security_officer.dm [hunk 1]
```diff
@@ -247,6 +247,16 @@ GLOBAL_LIST_INIT(available_depts, list(SEC_DEPT_ENGINEERING, SEC_DEPT_MEDICAL, S
 	uniform = /obj/item/clothing/under/rank/security/officer/nt
 	alt_uniform = null
 
+/datum/outfit/job/security/syndicate/gorlex
```
Comment (Fikou) [inline code/modules/jobs/job_types/security_officer.dm:250]: is there even a /datum/outfit/job/security/syndicate

**Comment by spockye** [comment]
I cant figure out this flicker chance thing, and i'd rather not have it go stale while i stare at it cluelessly.

**Comment by ZephyrTFA** [comment]
I have a better idea

**Comment by spockye** [comment]
https://user-images.githubusercontent.com/79304582/179510945-c96fb241-a934-4c59-9cd4-39651a628824.mp4

**Comment by spockye** [comment]
I'll finish this another time. Not too happy with it anyhow

**Pull request closed without merging**